.github/workflows/scripts/func_test_script.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# coding=utf-8
+
+set -mveuo pipefail
+
+pip install ../pulp_ansible
+pip install -r ../pulp_ansible/functest_requirements.txt
+
+pytest -v -r sx --color=yes --pyargs galaxy_ng.tests.functional
+
+cd ../pulp_ansible
+
+pytest -v -r sx --color=yes --pyargs pulp_ansible.tests.functional.cli
```

Comment:
I'm limiting to CLI tests while I'm investigating the issue with the other tests